### PR TITLE
initial implementation of wmts layer sample

### PR DESCRIPTION
**Pull request by tschie**
shows how to display a wmts layer as a basemap

### CODE REVIEW

File: src/main/java/com/esri/samples/ogc/wmts_layer/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<h1>WMTS Layer</h1>
+
+<p>Shows how to display a WMTS layer from a WMTS service.</p>
```
Comment (john0005) [inline src/main/java/com/esri/samples/ogc/wmts_layer/README.md]: Might replace first WMTS  with `Web Map Tile Service ` so people know what it stands for.

File: src/main/java/com/esri/samples/ogc/wmts_layer/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+<p><img src="WmtsLayer.png"/></p>
+
+<h2>How it works</h2>
+
+<p>To display a <code>WmtsLayer</code> from a <code>WmtsService</code>:</p>
+
+<ol>
+  <li>Create a <code>WmtsService</code> using the URL of the WMTS Service.</li>
+  </li>After loading the WmtsService, get the list of <code>WmtsLayerInfo</code>s from the service info:
```
Comment (john0005) [inline src/main/java/com/esri/samples/ogc/wmts_layer/README.md]: first `</li>` should be `<li>`

File: src/main/java/com/esri/samples/ogc/wmts_layer/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+<h2>How it works</h2>
+
+<p>To display a <code>WmtsLayer</code> from a <code>WmtsService</code>:</p>
+
+<ol>
+  <li>Create a <code>WmtsService</code> using the URL of the WMTS Service.</li>
+  </li>After loading the WmtsService, get the list of <code>WmtsLayerInfo</code>s from the service info: 
+  <code>service.getServiceInfo().getLayerInfos()</code></li>
+  <li>For the layer you want to display, get the layer ID using <code>getLayerInfos().get(0).getId()</code></li>
+  <li>Use the ID to create a WMTS Layer: <code>new WmtsLayer(serviceURL, layerID)</code><li>
```
Comment (john0005) [inline src/main/java/com/esri/samples/ogc/wmts_layer/README.md]: second`<li>` should be `</li>`

File: src/main/java/com/esri/samples/ogc/wmts_layer/WmtsLayerSample.java [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+      stage.show();
+
+      // create a map and add it to the map view
+      ArcGISMap map = new ArcGISMap();
+      mapView = new MapView();
+      mapView.setMap(map);
+
+      // create a WMTS service from a URL
+      String serviceURL = "http://sampleserver6.arcgisonline" +
+          ".com/arcgis/rest/services/WorldTimeZones/MapServer/WMTS";
```
Comment (john0005) [inline src/main/java/com/esri/samples/ogc/wmts_layer/WmtsLayerSample.java]: could put this on one line since its not pass 120

**Review by john0005 — CHANGES_REQUESTED** [review]
@tschie Always happy to review. Will try not to bug you on testing. 😸

**Comment by tschie** [comment]
Thanks for the review @john0005

**Pull request merged**